l10n_it_account/models/res_partner.py

```diff
@@ -0,0 +1,10 @@
+import re
+
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def format_vat_it(self, vat):
```

Comment:
Questo metodo viene chiamato da `base_vat` in https://github.com/odoo/odoo/blob/0268986b60477c7fb46d93f45f3355f546f0155c/addons/base_vat/models/res_partner.py#L545 quindi potresti aggiungere la dipendenza da `base_vat`?

Attualmente viene comunque chiamato dai test (è _coperto_) solo perché altri moduli dipendono da `base_vat` quindi viene comunque installato nell'ambiente di Travis.
Puoi verificarlo in locale installando solo questo modulo: questo metodo non viene mai chiamato.

Reply:
Non so se ho ben capito.

base_vat chiama questo modulo, non viceversa. Sarebbe una dipendenza se questo modulo chiamasse funzioni di base_vat.

Ora, la funzionalità del test non è disponibile se manca base_vat. Non so se è possibile specificare una dipendenza che valga solo per i test, o se a livello di test abbia senso controllare che ci sia base_vat e sono in quel caso eseguire il test.

Reply:
In generale una situazione del genere si risolverebbe con un modulo _ponte_ ma in questo caso mi pare eccessivo, quindi lasciamo pure così 👍🏻